src/critnib/critnib.c

```diff
@@ -650,6 +655,11 @@ static struct critnib_leaf *find_successor(struct critnib_node *__restrict n) {
         }
 
         n = n->child[nib];
```

Comment:
just asking - shouldn't we also have atomic_load_acquire here?

Reply:
@ldoray did it in the scope of #1143. See this [commit](https://github.com/oneapi-src/unified-memory-framework/pull/1143/commits/44e498193139bfed65b874619c24c06bc0196c52)